lib/rspec/matchers/built_in/include.rb

```diff
@@ -13,34 +13,50 @@ def initialize(*expected)
         # @return [Boolean]
         def matches?(actual)
           @actual = actual
-          perform_match(:all?, :all?)
+          if actual.respond_to?(:include?)
+            @divergent_items = excluded_from_actual
```

Comment:
Woops, realized my suggestion above would break things.  Updated:

``` ruby
def matches?(actual)
  @actual = actual
  @divergent_items = excluded_from_actual
  actual.respond_to?(:include) && @divergent_items.empty?
end

def does_not_match?(actual)
  @actual = actual
  @divergent_items = included_from_actual
  actual.respond_to?(:include) && @divergent_items.empty?
end

def excluded_from_actual(...)
  return [] unless actual.respond_to?(:include?)
  # ...
end